src/backend/opencl/CMakeLists.txt

```diff
@@ -1,5 +1,8 @@
 FIND_PACKAGE(OpenCL REQUIRED)
 
+OPTION(USE_CLBLAST "Build ArrayFire with the CLBlast BLAS library for the OpenCL backend" OFF)
+OPTION(USE_CLBLAS "Build ArrayFire with the clBLAS BLAS library for the OpenCL backend" ON)
```

Comment:
We do that with the CMAKE_BUILD_TYPE option. You can find the code [here](https://github.com/arrayfire/arrayfire/blob/devel/CMakeLists.txt#L38).